cmd/utils/nodecmd/dumpconfigcmd.go

```diff
@@ -120,22 +146,652 @@ func makeConfigNode(ctx *cli.Context) (*node.Node, klayConfig) {
+// a key index in the key store to an internal account representation.
+func makeAddress(ks *keystore.KeyStore, account string) (accounts.Account, error) {
+	// If the specified account is a valid address, return it
+	if common.IsHexAddress(account) {
+		return accounts.Account{Address: common.HexToAddress(account)}, nil
+	}
+	// Otherwise try to interpret the account as a keystore index
+	index, err := strconv.Atoi(account)
+	if err != nil || index < 0 {
+		return accounts.Account{}, fmt.Errorf("invalid account address or index %q", account)
+	}
+	logger.Warn("Use explicit addresses! Referring to accounts by order in the keystore folder is dangerous and will be deprecated!")
+
+	accs := ks.Accounts()
+	if len(accs) <= index {
+		return accounts.Account{}, fmt.Errorf("index %d higher than number of accounts %d", index, len(accs))
+	}
+	return accs[index], nil
+}
+
+func setTxPool(ctx *cli.Context, cfg *blockchain.TxPoolConfig) {
+	if ctx.GlobalIsSet(utils.TxPoolNoLocalsFlag.Name) {
+		cfg.NoLocals = ctx.GlobalBool(utils.TxPoolNoLocalsFlag.Name)
+	}
+	if ctx.GlobalIsSet(utils.TxPoolAllowLocalAnchorTxFlag.Name) {
+		cfg.AllowLocalAnchorTx = ctx.GlobalBool(utils.TxPoolAllowLocalAnchorTxFlag.Name)
+	}
+	if ctx.GlobalIsSet(utils.TxPoolDenyRemoteTxFlag.Name) {
+		cfg.DenyRemoteTx = ctx.GlobalBool(utils.TxPoolDenyRemoteTxFlag.Name)
+	}
+	if ctx.GlobalIsSet(utils.TxPoolJournalFlag.Name) {
+		cfg.Journal = ctx.GlobalString(utils.TxPoolJournalFlag.Name)
+	}
+	if ctx.GlobalIsSet(utils.TxPoolJournalIntervalFlag.Name) {
+		cfg.JournalInterval = ctx.GlobalDuration(utils.TxPoolJournalIntervalFlag.Name)
+	}
+	if ctx.GlobalIsSet(utils.TxPoolPriceLimitFlag.Name) {
+		cfg.PriceLimit = ctx.GlobalUint64(utils.TxPoolPriceLimitFlag.Name)
+	}
+	if ctx.GlobalIsSet(utils.TxPoolPriceBumpFlag.Name) {
+		cfg.PriceBump = ctx.GlobalUint64(utils.TxPoolPriceBumpFlag.Name)
+	}
+	if ctx.GlobalIsSet(utils.TxPoolExecSlotsAccountFlag.Name) {
+		cfg.ExecSlotsAccount = ctx.GlobalUint64(utils.TxPoolExecSlotsAccountFlag.Name)
+	}
+	if ctx.GlobalIsSet(utils.TxPoolExecSlotsAllFlag.Name) {
+		cfg.ExecSlotsAll = ctx.GlobalUint64(utils.TxPoolExecSlotsAllFlag.Name)
+	}
+	if ctx.GlobalIsSet(utils.TxPoolNonExecSlotsAccountFlag.Name) {
+		cfg.NonExecSlotsAccount = ctx.GlobalUint64(utils.TxPoolNonExecSlotsAccountFlag.Name)
+	}
+	if ctx.GlobalIsSet(utils.TxPoolNonExecSlotsAllFlag.Name) {
+		cfg.NonExecSlotsAll = ctx.GlobalUint64(utils.TxPoolNonExecSlotsAllFlag.Name)
+	}
+
+	cfg.KeepLocals = ctx.GlobalIsSet(utils.TxPoolKeepLocalsFlag.Name)
+
+	if ctx.GlobalIsSet(utils.TxPoolLifetimeFlag.Name) {
+		cfg.Lifetime = ctx.GlobalDuration(utils.TxPoolLifetimeFlag.Name)
+	}
+
+	// PN specific txpool setting
+	if utils.NodeTypeFlag.Value == "pn" {
+		cfg.EnableSpamThrottlerAtRuntime = !ctx.GlobalIsSet(utils.TxPoolSpamThrottlerDisableFlag.Name)
+	}
+}
+
+// getNetworkID returns the associated network ID with whether or not the network is private.
```

Comment:
```suggestion
// getNetworkId returns the associated network ID with whether or not the network is private.
```
Seems like a minor typo